### ISSUE TITLE
[doc] FeatureAlphaDropout not documented

### ISSUE THREAD

**Issue by SsnL** [pytorch/pytorch#9886]

**Comment by deepali-c** [comment]
`DistributedDataParallelCPU `needs to be added.

**Comment by SsnL** [comment]
@deepali-c thanks! added to the list.

**Comment by vadimkantorov** [comment]
@SsnL For softplus the docs are written and visible in the code: https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/activation.py#L577

So enabling the docs for the web probably should be easy (autoclass is there, but sth works wrong?)

**Comment by SsnL** [comment]
@vadimkantorov Yeah, likely something in doc building caused it to miss softplus.

**Comment by vadimkantorov** [comment]
@SsnL I found the reason:
https://github.com/pytorch/pytorch/blob/master/torch/nn/functional.py#L1109 is almost empty. At least ``` See :class:`~torch.nn.Softplus` for more details.``` should be put there.

**Comment by xuhdev** [comment]
Is this issue blocked? Are we still interested in resolving this issue?

**Comment by SsnL** [comment]
@xuhdev It is not blocked. And I am confused. Why would we not want to add & improve docs?

**Comment by xuhdev** [comment]
I've added the "good first issue". Hopefully this will attract some interest.

**Comment by pietern** [comment]
@SsnL `DistributedDataParallelCPU` can be removed from the list now that #22864 is merged.

**Comment by pdlive215** [comment]
Are there standards or templates for documentation? I'd like to submit a PR for `nn.functional.bilinear`, but existing documentation is inconsistent. Most `nn.functional` functions have parameters labeled as `Args`, but for `nn.functional.linear` the parameters are labeled as `Shape`.

Also, some of the documentation in `nn.functional` is simply a reference to `nn`. Do we want to replace these references with explicit comments? Is there a reason why the documentation isn't consistent for functions shared between `nn.functional` and `nn`? The comments are more extensive for `nn.linear` than for `nn.functional.linear`.

I'd be happy to clean up the `nn.functional` docs if there's a common template on which we agree.

**Comment by choidongyeon** [comment]
Just want to get an idea of the progress in this issue. Copied @SsnL's list and added what seems to be relevant PRs:

- [ ]  FeatureAlphaDropout
- [x]  nn.init.zeros_, nn.init.ones_[23145](https://github.com/pytorch/pytorch/pull/23145)
- [x]  bilinear missing doc content [24951](https://github.com/pytorch/pytorch/pull/24951)
- [x]  DistributedDataParallelCPU [22864](https://github.com/pytorch/pytorch/pull/22864)
- [x]  torch.as_strided [22842](https://github.com/pytorch/pytorch/pull/22842)
- [x]  F.softplus [27045](https://github.com/pytorch/pytorch/pull/27045)
- [ ]  torch.default_generator 

Am I missing anything? Would like to work on this and close out these issues.

**Comment by choidongyeon** [comment]
@SsnL where does one find torch.default_generator in the code? I understand that it returns the default CPU torch.Generator, but I can't seem to actually locate where to add the documentations, just a lot of ```from torch._C import default_generator```.

**Comment by shreyashub** [comment]
The issue is still open, are there any topics left?. Would like to contribute if possible

**Comment by vadimkantorov** [comment]
FeatureAlphaDropout docs are still not online. Same for `feature_dropout`

**Comment by mtsokol** [comment]
@vadimkantorov Hi! I've been recently looking for a `good first issue`.
Can I try contributing by adding these? Or is it taken?

**Comment by vadimkantorov** [comment]
I am not an active code contributor, but I don't think a concept of "taken" exists at all in PyTorch project. Please go ahead!

**Comment by Naveen-singla** [comment]
I would like to work on this project. Can u please help me with how to get started actually I am new to the open source contributing community

**Comment by harishsdev** [comment]
please let me know what is missing in this issue?

**Comment by nouranali** [comment]
are there any points not implemented yet since the issue is still open?

**Comment by axsseldz** [comment]
Hi! I'd be happy to work on any features or issues that haven't been implemented yet. I'll review what's still pending and submit a PR shortly(:

**Comment by Zhi0467** [comment]
Seems like the [PR](https://github.com/pytorch/pytorch/pull/138295) is not merged? @axsseldz

**Comment by TeahR** [comment]
I checked the current built docs locally and `torch.default_generator` does render under Random sampling, but only with the short annotation “Returns the default CPU torch.Generator.”

Would maintainers be interested in expanding this entry with a brief explanation of when the default generator is used, or should this issue be considered resolved?

**Comment by helomelo1** [comment]
Hey I would love to work on this issue... could you please tell which functionalities aren't documented?
